src/Chamilo/CoreBundle/Component/Utils/ChamiloApi.php

```diff
@@ -253,4 +253,40 @@ public static function getLanguageVar($text, $prefix = '')
 
         return $prefix.$text;
     }
+
+    /**
+     * Get the stylesheet path for HTML documents created with CKEditor.
+     *
+     * @return string
```

Comment:
Users can edit 2 css files now (document.css and editor_content.css) ?
Why this is needed?

Reply:
`editor_content.css` es cargado globalmente dentro de chamilo. Y también es cargado internamente por ckeditor para mostrar los estilos que pueden aparecer en la lista de estilos. Es necesario cargarlo por chamilo para que los bloques incrustados (como introduciones de herramientas) tengan los estilos definidos en este archivo.

![imagen](https://user-images.githubusercontent.com/1697880/41298879-d49787d2-6e27-11e8-9a4c-a453a3e71d9f.png) ![Styles](https://user-images.githubusercontent.com/1697880/41297975-fbea7c88-6e25-11e8-894f-59e9954471a9.png)
 
A esta lista se pueden agregar más según los requerimientos del tema de los usuarios. Los nuevos estilos serán agregados a este archivo. Ojo que estos estilos SÍ afectan a las hojas de estilo de chamilo, como base.css.

Cuando creas un documento, `document.css` (que importa a editor_content.css) es incluido dentro del archivo html que se generará. Aquí se pueden agregar otros estilos que sólo serán para los documentos HTML, y que NO afectan a chamilo. Mientras se escribe el archivo, el usuario va viendo los estilos de los documentos, pero éstos no afectan a chamilo, porque sólo le afectan internamente a ckeditor.

creando bloque | creando documento
-- | --
![imagen](https://user-images.githubusercontent.com/1697880/41299875-0d7651ee-6e2a-11e8-886a-8f71f0590c4e.png) | ![imagen](https://user-images.githubusercontent.com/1697880/41299744-c0124f3e-6e29-11e8-83f4-4b44ca31aa12.png)
blockquote de bootstrap | bloquote personalizado para documentos

Reply:
Esta buena la explicación, tal vez ponerlo en un wiki?

Reply:
Después del merge :smiley: